k8s/gvclient.go

```diff
@@ -32,26 +34,50 @@ type groupVersionClient struct {
 
 func (g *groupVersionClient) get(ctx context.Context, identifier resource.Identifier, plural string,
 	into resource.Object) error {
+	ctx, span := GetTracer().Start(ctx, "kubernetes-get")
+	defer span.End()
 	sc := 0
 	request := g.client.Get().Resource(plural).Name(identifier.Name)
 	if strings.TrimSpace(identifier.Namespace) != "" {
 		request = request.Namespace(identifier.Namespace)
 	}
 	bytes, err := request.Do(ctx).StatusCode(&sc).Raw()
+	span.SetAttributes(
+		attribute.Int("http.response.status_code", sc),
+		attribute.String("http.request.method", http.MethodGet),
+		attribute.String("server.address", request.URL().Hostname()),
+		attribute.String("server.port", request.URL().Port()),
+		attribute.String("url.full", request.URL().String()),
```

Comment:
Shouldn't this be part of the http.request attributes?
```suggestion
		attribute.String("http.request.url", request.URL().String()),
```

Reply:
Per the [Semantic Conventions for OpenTelemetry HTTP Spans](https://github.com/open-telemetry/semantic-conventions/blob/098db1ca510da01fe941f6c6308aad5000def910/docs/http/http-spans.md#http-client), they just list `url.full` (not 100% sure why a lot of them don't have the `http` prefix, but was trying to abide by them).

Reply:
Aha, good to know. Thanks!